src/hooks/api/getters/useFeaturesArchive/useFeaturesArchive.ts

```diff
@@ -9,10 +9,19 @@ export interface IUseFeaturesArchiveOutput {
     error?: Error;
 }
 
-export const useFeaturesArchive = (): IUseFeaturesArchiveOutput => {
+export const useFeaturesArchive = (
```

Comment:
I think I would prefer to separate this into a `useProjectArchive` hook instead of augmenting this hook with conditionals.

Reply:
Makes sense. Would `useProjectFeaturesArchive` be clearer? Who knows, we might have a project archive (archiving the projects themselves) in the future.